### PR TITLE
[receiver/hostmetrics] remove direction from process scraper 

### LINKED ISSUE #11819: `[receiver/hostmetrics/processscraper]`: remove `direction` attribute

### PR DESCRIPTION
**Pull request by codeboten**
Fix https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/11819

### CODE REVIEW

File: unreleased/hostmetrics-rm-direction-process.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+component: hostmetricsreceiver
+
+note: "Remove direction for paging metrics. The feature gate: `receiver.hostmetricsreceiver.removeDirectionAttributeProcessMetrics` can be set to apply the following"
+
+issues: [11819]
+
+subtext: |2
+    - `process.disk.io` will become:
+      - `process.disk.io.read`
+      - `process.disk.io.write`
```
Comment (djaglowski) [inline unreleased/hostmetrics-rm-direction-process.yaml:12]: #11866 ended up padding this text automatically, so the following should be equivalent:
```suggestion
subtext:
  - `process.disk.io` will become:
    - `process.disk.io.read`
    - `process.disk.io.write`

```

**Review by djaglowski — COMMENTED** [review]
LGTM. One nit noted.

File: unreleased/hostmetrics-rm-direction-process.yaml (hunk 1 above)
Comment (codeboten) [reply unreleased/hostmetrics-rm-direction-process.yaml]: @djaglowski 

```
FAIL: validate: yaml: line 10: found character that cannot start any token
make: *** [chlog-validate] Error 1
```

File: unreleased/hostmetrics-rm-direction-process.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+change_type: deprecation
+
+component: hostmetricsreceiver
+
+note: "Remove direction for paging metrics. The feature gate: `receiver.hostmetricsreceiver.removeDirectionAttributeProcessMetrics` can be set to apply the following"
+
+issues: [11819]
+
+subtext:
```
Comment (djaglowski) [inline unreleased/hostmetrics-rm-direction-process.yaml:9]:
```suggestion
subtext: |
```

Sorry, this is still needed to indicate a multiline string

Comment (codeboten) [reply unreleased/hostmetrics-rm-direction-process.yaml]: Thanks, I was just getting back to this :D

**Review by dmitryax — APPROVED** [review]

**Pull request merged**